media-sound/pulsemixer/pulsemixer-1.5.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="ncurses"
+
+inherit distutils-r1
+
+DESCRIPTION="cli and curses mixer for pulseaudio"
+HOMEPAGE="https://github.com/GeorgeFilipkin/pulsemixer"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/GeorgeFilipkin/${PN}"
+else
+	SRC_URI="https://github.com/GeorgeFilipkin/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND=">=media-sound/pulseaudio-12.2"
```

Comment:
Is Version 12.2 as a minimum correct? Github states `1.0`?

Reply:
`=pulseaudio-12.2` was the oldest PA in-tree when I opened this so I figured that was good enough. Is it necessary to check a package against dependency versions that aren't in-tree because an upstream says it's compatible?

Reply:
Adjusted as per IRC.